### PR TITLE
Removing obsolete package from metricbeat

### PR DESCRIPTION
**Pull request by amitkanfer**
github.com/pkg/errors is [deprecated](https://github.com/pkg/errors) since 2021.
Furthermore, it's at least 100% worse in terms of performance compared to the native errors and fmt packages, (probably due to the fact every error log comes with a [stack trace](https://github.com/pkg/errors/blob/master/errors.go#L115))

Another interesting benchmark: https://g4s8.wtf/posts/go-errors-performance/

### CODE REVIEW

File: metricbeat/module/docker/healthcheck/data.go
Comment (amitkanfer) [inline metricbeat/module/docker/healthcheck/data.go:46]: Special attention here

**Review by cmacknz — APPROVED** [review]
Everything LGTM, the one place the error was unused feels like it is a bug that could be improved but within the scope of what this PR is trying to do it could also be left as is.

File: metricbeat/module/docker/healthcheck/data.go
Comment (amitkanfer) [reply metricbeat/module/docker/healthcheck/data.go:45]: ok, deleting. will open a follow up issue to fix this.

**Comment by amitkanfer** [comment]
Team,
We're making an exception and not going to wait for approvals from all CODEOWNERS for this "find and replace" type of PRs. @cmacknz 's review will be enough.

**Pull request merged**
